test/test_command.py

```diff
@@ -66,3 +66,110 @@ def test_parse_commands_failure(capsys, command, sys_exit_code, sys_exit_message
 
     captured_stderr = capsys.readouterr().err
     assert sys_exit_message in captured_stderr
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        # Select no target - only one available
+        {"command": ["compile", "-o", "OUTPUTFILE", "INPUTFILE"],
+         "input_data": """otk.version: "1"
```

Comment:
(quick drive-by) I see this 
"""otk.version: "1"

otk.target.osbuild.name:
  pipelines:
    - "test"
"""
a lot - could it be extracted into a var and just used as a var like `minimal_otk` and then (maybe) `minimal_otk + "whatever needs adding" - this would probably make the reading of this easier.

Alternatively there is test/test_examples.py and test/data/error, could some be put there (I guess there is no way to manipulate what `-t` is used there so mabe that won't work).

Reply:
I thought so too - not sure if this looks better - I'll check again
Second option won't work easily, that's right